modules/mono/editor_templates/plugin.cs

```diff
@@ -0,0 +1,18 @@
+#if TOOLS
+using _BINDINGS_NAMESPACE_;
+using System;
+
+[Tool]
+public class _CLASS_ : _BASE_
```

Comment:
Missing the `partial` modifier here.

```suggestion
public partial class _CLASS_ : _BASE_
```